src/test/java/io/appium/java_client/android/AndroidDriverTest.java

```diff
@@ -118,7 +118,7 @@
 
     @Test public void getDeviceTimeTest() {
         String time = driver.getDeviceTime();
-        assertTrue(time.length() == 28);
```

Comment:
Import statement can also be removed.

Reply:
which import?

Reply:
https://github.com/appium/java-client/pull/939/files/97a1c09c26bdfdfae23accd2689f02c326caf8b3#diff-4cea3ff81e0a1323e7aeb653d2e4a366L26

Reply:
it's still used by `isAppInstalledTest` and `lockTest`. Codacy usually highlights such cases automatically.